Experiment.xcodeproj/project.pbxproj

```diff
@@ -311,6 +312,27 @@
 		};
 /* End PBXResourcesBuildPhase section */
 
+/* Begin PBXShellScriptBuildPhase section */
+		19F2A56C28FDE17C002DF58F /* ShellScript */ = {
```

Comment:
This is to fix the build failed issue of `xframework that cannot be found`  after running `carthage update --use-xcframeworks`.
The changes are coming from adding the script `/usr/local/bin/carthage copy-frameworks
` under the Build Phases tab.
For reference. https://kevinle.medium.com/step-by-step-guide-on-using-carthage-dependency-manager-a29c15f9a1ac.

Reply:
@bgiori Can you confirm if we need this?

Reply:
I did a fresh clone of the repository, ran `carthage update --use-xcframeworks` opened the project in xcode and built it successfully without the build phase in question. 

What carthage version are you using? `carthage version`

Reply:
@bgiori  I'm using `0.38.0` how about yours?

Reply:
Yeah, same. Do you have the exact output for the error which this change fixes?